backend/src/gpml/db/submission.sql

```diff
@@ -40,9 +40,11 @@ SELECT json_build_object(
 
 -- :name detail :? :1
 -- :doc get detail of submission
-SELECT *
-from :i:table-name
-where id = :id::integer;
+SELECT s.email AS created_by_email, t.*
```

Comment:
why do we need to rename it?
and why at this sql query level and not in gpml.[db|handler].submission clj namespace ?

Reply:
I wanted to let the `created_by` (stakeholder ID to be available), in case we end up using it, for instance to link to the stakeholder detail page from the submission info, for instance..